### DISCUSSION TITLE
pictures (links) in wiki are not shown

### DISCUSSION THREAD

**Discussion by OE1MWW** [egzumer/uv-k5-firmware-custom#154] [General]
for example in 'Spectrum analyzer' and on other entries ;-(

**Comment by egzumer** [comment]
there is some issue with Github, they sometimes show up, sometimes not

**Comment by kittenarmy** [comment]
All the images appear to me as private -- images prefixed with [private image](https://private-user-images.githubusercontent.com/) have access controls you can set in the original repo. 
https://www.reddit.com/r/github/comments/13foas2/github_no_longer_allows_image_uploads_to_be_public/

**Comment by egzumer** [comment]
https://github.com/orgs/community/discussions/77236

**Comment by kittenarmy** [comment]
Temporary workaround: if I edit the wiki to view the original urls they embed after a refresh.

**Comment by egzumer** [comment]
They disapper after very short time.

**Comment by kittenarmy** [comment]
So frustrating!!
